$NetBSD: patch-exports_Makefile,v 1.1 2026/02/24 23:09:02 thor Exp $

Fix SONAME for FIXED_LIBNAME (adding major version suffix in Makefile.system).

--- exports/Makefile.orig	2026-01-15 22:57:26.000000000 +0000
+++ exports/Makefile
@@ -140,7 +140,7 @@ libgoto_hpl.def : $(GENSYM)
 
 ifeq ($(OSNAME), Darwin)
 ifeq ($(FIXED_LIBNAME),1)
-INTERNALNAME = $(LIBPREFIX).dylib
+INTERNALNAME = $(LIBDYNNAME)
 else
 INTERNALNAME = $(LIBPREFIX).$(MAJOR_VERSION).dylib
 endif
@@ -181,7 +181,7 @@ FEXTRALIB += -lm
 EXTRALIB += -lm
 else
 ifeq ($(FIXED_LIBNAME),1)
-INTERNALNAME = $(LIBPREFIX).so
+INTERNALNAME = $(LIBSONAME)
 else
 INTERNALNAME = $(LIBPREFIX).so.$(MAJOR_VERSION)
 endif
